src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        String randomColor = ColorSupplier.getRandomColor();
+        Random random = new Random();
+        int randomNumber = random.nextInt(100) + 1;

```

Comment:
100 is a magic number.
direct use of MAgic numbers should be avoided.
consider making it constants with self-descriptive name.
```suggestion
        int randomNumber = random.nextInt(100) + 1;

```